number_theory/numeric_unittest.cpp

```diff
@@ -0,0 +1,42 @@
+#include <stdint.h>
+
+#include <cmath>
+#include <limits>
+#include <random>
+
+#include <gtest/gtest.h>
+
+#include "number_theory/numeric.h"
+
+using namespace tql::number_theory;
```

Comment:
Instead of this, we should wrap the whole file with namespace tql::number_theory with a nested namespace statement just like we do in other files.